cmd/ui/src/ducks/explore/reducer.ts

```diff
@@ -51,10 +51,34 @@ const graphDataReducer = (state = initialGraphDataState, action: types.GraphActi
         } else if (action.type === types.TOGGLE_TIER_ZERO_NODE) {
             const systemTags = state.chartProps.items[action.nodeId].data.system_tags;
             // remove the tier zero tag from the node
-            if (systemTags === 'admin_tier_0') {
-                draft.chartProps.items[action.nodeId].data.system_tags = '';
+            if (systemTags.includes('admin_tier_0')) {
```

Comment:
I appreciate that you considered the handling of both tags here in various situations. We can accomplish something similar by using the `replace` function which makes this logic more concise and clear. 

```typescript
        } else if (action.type === types.TOGGLE_TIER_ZERO_NODE) {
            const systemTags = state.chartProps.items[action.nodeId].data.system_tags;
            // remove the tier zero tag from the node
            systemTags.replace(TIER_ZERO_TAG, "").trim();
        } else if (action.type === types.TOGGLE_OWNED_OBJECT_NODE) {
            const systemTags = state.chartProps.items[action.nodeId].data.system_tags;
            // remove the owned object tag from the node
            systemTags.replace(OWNED_OBJECT_TAG, "").trim();
        }
```

Here is some example usage from trying this out in the browser console:

```
let dualTag = "admin_tier_0 owned"
> undefined
dualTag.replace("owned", "").trim()
> "admin_tier_0"
let singleTag = "admin_tier_0"
> undefined
singleTag.replace("owned", "").trim()
> "admin_tier_0"
let emptyTag = ""
> undefined
emptyTag.replace("owned", "").trim()
> ""
let flippedDualTag = "owned admin_tier_0"
> undefined
flippedDualTag.replace("owned", "").trim()
> "admin_tier_0" 
```

Reply:
Great advice! Will look in to this. Do we need to consider multiple dangling spaces i.e. in the future if there is three or more system_tags? Ex:
```
let tripleTag = "admin_tier_0 some_tag owned"
> undefined
tripleTag.replace("some_tag", "").trim()
> "admin_tier_0  owned" // Two spaces
tripleTag.replace("some_tag", "").trim().replace(/ {2,}/g, " ")
> "admin_tier_0 owned" // Removed extra space in middle
```

Reply:
Hmm, yea that is a good point! Handling that scenario would be nice in case we did have more tags like you mention. You are welcome to handle that here as you wish but if you are using regex pattern matching maybe leave a comment explaining what it is doing